elyra/pipeline/validation.py

```diff
@@ -428,6 +429,20 @@ async def _validate_generic_node_properties(self, node: Node, response: Validati
                         resource_value=resource_value,
                         response=response,
                     )
+            for resource_vendor in ["gpu_vendor"]:
+                vendor = node.get_component_parameter(resource_vendor)
+                if not is_valid_kubernetes_device_plugin_name(vendor):
```

Comment:
Function `is_valid_kubernetes_device_plugin_name` is returning `False` if `vendor` is not set, which causes two tests in elyra/tests/pipeline/test_validation.py to fail:
 - `test_valid_node_property_pipeline_filepath`
 - `test_invalid_node_property_structure`

```
'message': 'Property is not a valid resource vendor name.', 'severity': 1, 'source': 'Elyra Pipeline Validation Service', ...}, {'data': {'nodeID': '1e7ffe03-54b8-412b-8ce0-2707181fd5e8', 'nodeName': 'notebookD.ipynb', 'propertyName': 'gpu_vendor', 'value': None},
```

Since the behavior of `is_valid_kubernetes_device_plugin_name` is correct (an empty vendor name is not valid), we need to only call the function if a value was provided. (Validation is performed before the processor is invoked and therefore the 'default' value `nvidia.com/gpu` hasn't been assigned yet.